app/views/api_docs/vendor_api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,10 @@
+## 3rd June
+
+`HESAITTData` now includes the following optional fields:
+
+- `other_disability_details` will contain the candidate’s description of their disability if they selected “Other”. This corresponds to HESA disability code `96`
+- `other_ethnicity_details` will contain the candidate’s description of their ethnicity if they selected “Other”. This corresponds to HESA ethnicity code `80`
```

Comment:
just to double check `80` covers all values under `OTHER_ETHNIC_BACKGROUNDS`?

Reply:
Ah good catch, this is no longer the case. I'll remove the mention of the code from here